src/scss/_variables.scss

```diff
@@ -78,3 +78,11 @@ $header-height: 44px;
     backdrop-filter: none;
   }
 }
+
+@mixin draggable-hover-border {
+  outline: 1px solid #ddd;
+
+  @include phone-portrait {
+    outline: none;
+  }
+}
```

Comment:
I have extracted out the styles for the draggable item's hover border into a mixin that is reused by the subclass pseudo-element. I'm unsure whether this is correct. Maybe the styles should be repeated in the psuedo-element instead? Or possibly the mixin should be moved to **InventoryItem.m.scss** as it only relates to the draggable item?